core/kazoo_attachments/src/aws/kz_aws.erl

```diff
@@ -298,16 +299,8 @@ configure(#aws_config{} = Config) ->
 
 -spec get_metadata_credentials(aws_config()) -> {'ok', #metadata_credentials{}} | {'error', any()}.
```

Comment:
How about not creating this function and using `get_credentials_from_metadata/1` directly?
Also, looks like there's no longer a need to `application:set_env(kazoo_attachments, metadata_credentials, ...)`